content/docs/command-reference/repro.md

```diff
@@ -179,10 +179,10 @@ up-to-date and only execute the final stage.
   corresponding pipelines, including the target stages themselves. This option
   has no effect if `targets` are not provided.
 
-- `--pull` - [pulls](/doc/command-reference/pull) dependencies and outputs
-  involved in the stages being reproduced, if they are found in the
-  [default remote storage](/doc/command-reference/remote/default). Note that it
-  tries the local run-cache first (unless `--no-run-cache` is also used).
+- `--pull` - downloads dependencies and outputs in the stages being reproduced
+  from the [default remote storage](/doc/command-reference/remote/default) (see
+  `dvc pull`) based on the run-cache. Note that this doesn't include initial
+  pipeline data sources (never found in the run-cache).
```

Comment:
Is this better @pmrowla ? Thanks

Reply:
p.s. it's a kind of confusing option. Why is `--pull` specifically related to run-cache? 🤷 But anyway, when #4742 is addressed this should be taken care of.

Reply:
likely just because the original request was just for a method to pull run-cache+outputs for specific (run-cached) stages: https://github.com/iterative/dvc/issues/4223

Reply:
I think we should probably not use `dependencies` here at all. Maybe just say something like this?
`[pulls](...) outputs for any stages which restored from [run-cache](/doc/user-guide/experiment-management#automatic-log-of-stage-runs-run-cache) during pipeline reproduction. Outputs will be pulled from the [default remote storage](...).`

Reply:
Good call. Updated 👍